Gemfile

```diff
@@ -10,8 +10,7 @@ def vanagon_location_for(place)
   end
 end
 
-gem 'vanagon', *vanagon_location_for(ENV['VANAGON_LOCATION'] || 'git@github.com:puppetlabs/vanagon#master')
-gem 'packaging', :github => 'puppetlabs/packaging', :branch => 'master'
-gem 'artifactory'
+gem 'vanagon', *location_for(ENV['VANAGON_LOCATION'] || '~> 0.15')
+gem 'packaging', *location_for(ENV['PACKAGING_LOCATION'] || '~> 0.99')
```

Comment:
How will this work if PACKAGING_LOCATION is set to `0.99.2`?

Reply:
oof, good catch. location_for needs to be updated to not hardcode the vanagon repo when looking for a tag

Reply:
Can I just replace that line with `[place, { :require => false }]` ?

Reply:
and remove conditionals? Or what do we want it to do?

Reply:
That seems to work!

Reply:
I think we still need the conditionals so we can run from different forks or local checkouts